nav2_planner/src/planner_server.cpp

```diff
@@ -111,6 +112,16 @@ PlannerServer::on_configure(const rclcpp_lifecycle::State & state)
     planner_ids_concat_ += plugin_ids_[i] + std::string(" ");
   }
 
+  get_parameter("expected_planner_frequency", expected_planner_frequency_);
+  if (expected_planner_frequency_ <= 0)
+  {
+    RCLCPP_FATAL(
```

Comment:
I think this should just be a warning. I think 0 or negative values could be a way to disable this feature. I'd warn that using 0 or negative disables the feature and then use `<= 0` as a check of whether to do this